applications/nrf5340_audio/broadcast_source/main.c

```diff
@@ -325,8 +325,8 @@ int main(void)
 
 	size_t ext_adv_size = 0;
 	size_t per_adv_size = 0;
-	const struct bt_data *ext_adv = NULL;
-	const struct bt_data *per_adv = NULL;
+	static const struct bt_data *ext_adv;
+	static const struct bt_data *per_adv;
```

Comment:
Suggest we take this out. This is another PR.

Reply:
Created another PR for that:
https://github.com/nrfconnect/sdk-nrf/pull/14217